### DISCUSSION TITLE
Are there any instructions on using IfcCSV as a standalone script ?

### DISCUSSION THREAD

**Discussion by Jimmy0101010** [IfcOpenShell/IfcOpenShell#3266] [General]
Hi, 

I have an IFC model that I would like to export to CSV, are there any instructions on using IfcCSV as a standalone script ?

**Comment by Jimmy0101010** [comment]
@Moult

**Comment by Moult** [comment]
There is now :) https://github.com/IfcOpenShell/IfcOpenShell/commit/47c6c2ce9aad493c700a4f45cf9d804d0ed07ff9

**Reply by Jimmy0101010** [reply]
Thanks Dion!
